src/components/Dashboard/ThisWeek/NewsletterMenu/MenuItems/LogNewsLetter/LogNewsletter.tsx

```diff
@@ -31,28 +56,280 @@ const CloseButton = styled(IconButton)(({ theme }) => ({
+                        label={t('Completed Date')}
+                        okLabel={t('OK')}
+                        todayLabel={t('Today')}
+                        cancelLabel={t('Cancel')}
+                        clearLabel={t('Clear')}
+                        InputProps={{
+                          startAdornment: (
+                            <InputAdornment position="start">
+                              <IconButton>
+                                <CalendarIcon />
+                              </IconButton>
+                            </InputAdornment>
+                          ),
+                        }}
+                      />
+                    </Grid>
+                    <Grid item xs={6}>
+                      <TimePicker
+                        value={completedAt}
+                        onChange={(date): void =>
+                          setFieldValue('completed', date)
+                        }
+                        clearable
+                        fullWidth
+                        autoOk
+                        inputVariant="outlined"
+                        label={t('Completed Time')}
+                        okLabel={t('OK')}
+                        todayLabel={t('Today')}
+                        cancelLabel={t('Cancel')}
+                        clearLabel={t('Clear')}
+                        InputProps={{
+                          startAdornment: (
+                            <InputAdornment position="start">
+                              <IconButton>
+                                <ClockIcon />
+                              </IconButton>
+                            </InputAdornment>
+                          ),
+                        }}
+                      />
+                    </Grid>
+                  </Grid>
+                </Grid>
+              </LogFormControl>
+              <Grid item xs={12}>
+                <LogFormControl>
+                  <LogFormLabel>{t('Comment')}</LogFormLabel>
+                  <LogTextField
+                    value={commentBody}
+                    onChange={(event) => changeCommentBody(event.target.value)}
+                    fullWidth
+                    multiline
+                    inputProps={{ 'aria-label': 'Comment' }}
+                    variant="outlined"
+                  />
+                </LogFormControl>
+              </Grid>
+            </Grid>
+          </DialogContent>
+          <DialogActions>
+            <Button disabled={isSubmitting} onClick={handleClose}>
+              {t('Cancel')}
+            </Button>
+            <Button
+              size="large"
+              variant="contained"
+              color="primary"
+              disabled={!isValid || isSubmitting}
+              type="submit"
+            >
+              {creating && (
+                <>
+                  <CircularProgress color="primary" size={20} />
+                  &nbsp;
+                </>
```

Comment:
Any of this actually needed?
```suggestion
                <CircularProgress color="primary" size={20} />
```

Reply:
It provides a little bit of spacing between the progress circle and the ```save``` text, but maybe that could be done with padding/margin?

Reply:
🤷‍♂️ Either is fine. `&nbsp;` feels a little hackier...

Reply:
I agree, I'll go ahead and use margin